Makefile

```diff
@@ -201,10 +201,8 @@ test-integration:
 	$(CONTAINER_CMD) --security-opt label=disable --cap-add=cap_mknod -v $(CURDIR):$(CONTAINER_GOSRC) -w $(CONTAINER_GOSRC) $(SKOPEO_CIDEV_CONTAINER_FQIN) \
 		$(MAKE) test-integration-local
 
-
-# Intended for CI, assumed to be running in quay.io/libpod/skopeo_cidev container.
-test-integration-local: bin/skopeo
-	hack/warn-destructive-tests.sh
+# Expects binary in $PATH, so better to depend on install-binary target
+test-integration-local: install-binary
```

Comment:
This loses the effect of `warn-destructive-tests`.

Reply:
Right, so do we need `warn-destructive-tests` at all? If we do need them locally, do we also need them in CI environments where the tests are run on a fresh VM ?

Reply:
Ideally, tests don't modify the host system beyond temporary changes.  But AFAIK the CI context doesn't care about `warn-destructive-tests`, it happily goes and modifies the system w/o asking. This is only used for humans.